app/services/creators/other_incomes_creator.rb

```diff
@@ -56,5 +60,13 @@ def assessment
     def normalize(name)
       name.underscore.tr(" ", "_")
     end
+
+    def other_incomes
+      @other_incomes ||= JSON.parse(@other_incomes_params, symbolize_names: true)[:other_incomes]
```

Comment:
**nit**, for rubocop future proofing, and my preferred mechanism
```suggestion
      @other_incomes ||= JSON.parse(@other_incomes_params, symbolize_names: true).fetch(:other_incomes, nil)
```

Reply:
Thanks 👍 have made suggested change